### ISSUE TITLE
Diverts running over into choices issue 

### ISSUE THREAD

**Issue by madquills** [inkle/inky#9]
Like so: "You’ll see. But first — my Bonsai. 1: I love plants." 

Where it should be: 

```
You'll see, but first - my Bonsai -> plants 

= plants 

* I love plants 

```
For example.

**Comment by joethephish** [comment]
Hmm, sorry, not exactly sure what the problem is. Are you saying you need it to pause after "my Bonsai"? Or are you saying you need a line break, or...?

**Comment by madquills** [comment]
Ah sorry if that wasn't clear. So the line is being 'played' as including the first player dialogue choice after the divert. 

So Inky is currently having the NPC say: 

_"Ah ya mumu, I mean my bonsai. 1: Ok. How are they?"_ 
(all on the same line)

When the '1: Ok, how are they?" is actually the first player choice after a divert, so: 

=== knot === 
* blah
Ah ya mumu, I mean my bonsai -> bonsai 

= bonsai 

*  Ok. How are they?
* choice 2 
* choice 3

**Comment by joethephish** [comment]
Ah! So you're not seeing this?

![image](https://cloud.githubusercontent.com/assets/858967/16201130/2332cc2a-3708-11e6-9317-bff97d17aa22.png)

That's an annoying if it's working on mine but not yours, urgh :-/

**Comment by madquills** [comment]
When I recreate it like that it works. Here's what's happening to me,
rejigged the flow so I could fit it in a screenshot:



On Mon, Jun 20, 2016 at 4:58 PM Joseph Humfrey <notifications@github.com>
wrote:

> Ah! So you're not seeing this?
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/858967/16201130/2332cc2a-3708-11e6-9317-bff97d17aa22.png>
>
> That's an annoying if it's working on mine but not yours, urgh :-/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/inkle/inky/issues/9#issuecomment-227186099>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ATHn_dkOEtWTUpifz80HrSaBH04oMY2jks5qNri-gaJpZM4I51N6>
> .
> [image: Screen Shot 2016-06-20 at 17.02.28.png]
>
> --

*Danny Salfield Wadeson • *Producer/Writer

www.oddtales.net
+44 (0) 7853 255 718
@madquills

**Comment by joethephish** [comment]
Hrm, screenshot didn't make it through, sorry! If you attached it, I guess github doesn't support it (also, might want to be careful, I just edited your phone number out of your comment, which must've been on your sig!)

**Comment by madquills** [comment]
Here's the direct screenshot :) ![screen shot 2016-06-20 at 17 02 28](https://cloud.githubusercontent.com/assets/20047869/16202588/30e0dab4-370e-11e6-81c5-e505e5346734.png)

**Comment by joethephish** [comment]
Hmm, was the file originally written on Windows by any chance...?

**Comment by joethephish** [comment]
Found and fixed, in c4eef0c26e50a905fac731b1e1848caa3dddbe4a. Will make it into next build of inky...

**Comment by madquills** [comment]
Awesome, thanks.

On Tue, Jun 21, 2016 at 8:10 AM Joseph Humfrey <notifications@github.com>
wrote:

> Found and fixed, in c4eef0c
> <https://github.com/inkle/inky/commit/c4eef0c26e50a905fac731b1e1848caa3dddbe4a>.
> Will make it into next build of inky...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/inkle/inky/issues/9#issuecomment-227358399>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ATHn_YFjnR2J1gwtliLSeTGKnyhwuWZ_ks5qN45egaJpZM4I51N6>
> .
>
-- 

*Danny Salfield Wadeson • *Producer/Writer

www.oddtales.net
+44 (0) 7853 255 718
@madquills
